services/npm/npm-last-update.service.js

```diff
@@ -0,0 +1,96 @@
+import Joi from 'joi'
+import dayjs from 'dayjs'
+import { optionalUrl } from '../validators.js'
+import {
+  BaseJsonService,
+  InvalidParameter,
+  pathParam,
+  queryParam,
+} from '../index.js'
+import { formatDate } from '../text-formatters.js'
+import { age as ageColor } from '../color-formatters.js'
+
+const queryParamSchema = Joi.object({
+  registry_uri: optionalUrl,
+  version: Joi.string(),
+}).required()
+
+const updateResponseSchema = Joi.object({
+  time: Joi.object({
+    created: Joi.string().required(),
```

Comment:
Can we remove the test for `created`? i dont see we use it here.

Reply:
Yeah, we can remove `created`

Reply:
removed as part of https://github.com/badges/shields/pull/10641/commits/d39c4b54697d0582f96e7f69843d18a782d4a2ad